packages/checkout/text-input/text-input.tsx

```diff
@@ -26,6 +26,10 @@ interface TextInputProps
 	autoComplete?: string;
 	onChange: ( newValue: string ) => void;
 	onBlur?: ( newValue: string ) => void;
+	inputProps?: Omit<
```

Comment:
I'd recommend not having this extra level of inputProps. Ideally, we should use the component with
`<TextInput type="number" max="" min="" />` for a more intuitive API. Just like [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/text-control/index.js).

Reply:
I suppose this is better, the extra level of props was kind of pointless now that I think about it. Cheers 👏🏼